app/assets/javascripts/controllers/dialog_editor/dialog_editor_controller.js

```diff
@@ -65,7 +65,7 @@ ManageIQ.angular.app.controller('dialogEditorController', ['$window', 'miqServic
     translateResponderNamesToIds(dialog.content[0]);
 
     if (requestDialogAction() === 'copy') {
-      dialog.label = dialog.content[0].label = "Copy of " + dialog.label;
+      dialog.label = dialog.content[0].label = __('Copy of ') + dialog.label;
```

Comment:
I deliberately left this string untranslated, since it's going to be a label of a copied dialog and as such the dialog label will be rendered to all users in all locales as it was saved / created, i.e. I did not want to make any assumptions about the language, that the user wishes to use when creating a dialog copy.

Reply:
Makes sense. Closing. Thanks